UNLV-CS472-672/2024-S-GROUP5-Munch
# Add Backend Script for Running Unit Tests

This is related to issue # _replace or delete if N/A_

## Description / Changes Made
- Created a script that will test all backend tests in the unit_tests directory to be made soon.

## How to Test

1. _Provide step-by-step instructions on how to test the changes in this pull request for the reviewers. Include any specific setups or configurations needed._
2. Set up directory in backend directory named "unit_tests.py"
3. Create your test file, the name should be "test_<yourtestname>.py
4.  Run server.py using python3 server.py and python3 run_tests.py to run all test cases and receive coverage

## Checklist
